DEVELOPER.md

```diff
@@ -11,10 +11,14 @@ The project uses the following main frameworks:
 ## Preparations
 
 1. Install packages: `yarn`
-1. Compile: `npx hardhat compile`
-1. Generate typechain artifacts: `npm run generate:typechain`
-1. Run unit tests: `npx hardhat test`
-1. Run coverage test: `npm run run:coverage`
+2. Compile: `npx hardhat compile`
```

Comment:
there's no need to increment by hand. It's done automatically if you open the rendered readme. Same goes for all these number changes

Reply:
Great to know.